nupic/bindings/algorithms.i

```diff
@@ -1142,358 +778,9 @@ inline PyObject* generate2DGaussianSample(nupic::UInt32 nrows, nupic::UInt32 nco
 }
 
 //--------------------------------------------------------------------------------
-// Optimizations for FDRCSpatial2
+// Optimizations for FDR
```

Comment:
> Why can't we delete this whole section?

We cannot because the remaining `getSegmentActivityLevel` and `isSegmentActive` functions are currently used by TP (https://github.com/numenta/nupic/blob/master/nupic/research/TP.py#L36)